.github/workflows/npm-test.yml

```diff
@@ -3,7 +3,7 @@ name: npm-test
 on: workflow_call
 jobs:
   build:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

Comment:
Could we pin this (and the other) to match the `Dockerfile`? `python:3.10` is based on Debian 12. I don't see a Debian runner for GitHub Actions, but Debian 12 corresponds to Ubuntu 22.04 LTS per https://askubuntu.com/a/445496.

Ideally the `Dockerfile` would carry an admonition to update GitHub Actions runners whenever the base image is changed.

Reply:
I think python:3.10 is [based on Debian 11](https://hub.docker.com/layers/library/python/3.10/images/sha256-a4fbd0cbe6e333ed05a2af41d0771aecc90ad804fe8a58023c52265ad2bfea36)

Reply:
Should we be using 3.10-bookworm?